README.md

```diff
@@ -41,87 +41,94 @@
+1. Install `astro` by running the following command:
 
-Before running please keep in mind that you'll need to set the following env variable for passing table objects
-between tasks:
+    ```shell script
+    pip install astro-projects
+    ```
 
-```shell script
-AIRFLOW__CORE__ENABLE_XCOM_PICKLING=True
-```
+    Alternatively, you can add `astro-projects` to your `requirements.txt` file.
+
+2. Set the following environment variable so that `astro` can pass table objects between tasks:
+
+    ```shell script
+    AIRFLOW__CORE__ENABLE_XCOM_PICKLING=True
+    ```
 
 #  Using Astro as a SQL Engineer
 
 ## Schemas
 
 
-By default, our system will create a schema called `tmp_astro` in any database where astro runs, but we also realize that this system works with
-two core assumptions. The first assumption is that the data engineer running airflow can create schemas on the fly, and the second is that the user
-creating the schema will be the only user adding/removing from said schema.
+By default, running `astro` for a given database creates a schema called `tmp_astro` in the database. This default behavior assumes that you have permissions to create schemas on the fly, and that there is only one user adding/removing from this schema.
 
-For production usage we recommend that `astro` users work with their DBAs to create shared schemas where they can put their temporary tables. These schemas
+For production usage, we recommend that you work with your database administrator to create a shared schema for temporary tables. This schema
 can be shared across multiple users, but should be created with security in mind (e.g. don't place high security data in a shared schema).
 
-Once this schema is created, the Airflow admin can set the schema by setting the `AIRFLOW__ASTRO__SQL_SCHEMA` env variable, or setting the following in their
-`airflow.cfg`
+Once you create this schema, an Airflow admin can replace `tmp_astro` by setting `AIRFLOW__ASTRO__SQL_SCHEMA="<new-temp-schema>"`, or by setting the following in `airflow.cfg`:
 
 ```bash
 [astro]
-sql_schema=<your schema here>
+sql_schema=<new-temp-schema>
 ```
 
 ## Setting up SQL files
 
-When writing out a SQL DAG using `astro`, you can think of each SQL file as its own task. So for example if you wanted
-to aggregate orders, aggregate customers, and then join customers and orders you could have
+When writing out a SQL DAG using `astro`, you can think of each SQL file as its own task. For example, if you wanted
+to aggregate orders, aggregate customers, and then join customers and orders you could havesh
+
 ```
 |
 -- customers_table.sql
 -- orders_table.sql
 -- join_customers_and_orders.sql
 ```
 
-In each of these SQL files, you can create a table by having a standard `SELECT` statement (we will handle all of the
-temporary tables for you).
+In each of these SQL files, you can create a table by having a standard `SELECT` statement. `astro` handles creating all of the temporary tables.
```

Comment:
Not clear what table you're creating here. Is it the table that becomes our `output_table` in later suggestions?

Reply:
Make it clear that the table itself is automatically generated and can be referenced in other SQL queries by the data dependency alone